mysql-plugin/widgets/Mysql-batchsource.json

```diff
@@ -107,9 +107,80 @@
         }
       ]
     },
+    {
+      "label": "SSL",
+      "properties": [
+        {
+          "label": "Use SSL",
+          "name": "useSSL",
+          "widget-type": "select",
+          "widget-attributes": {
+            "default": "If available",
+            "values": ["No", "If available", "Require"]
```

Comment:
'No' and 'Require' seem like an odd pair. I think it's better to have:

'If Available', 'Yes', 'No'